integration-tests/src/test/java/org/zowe/apiml/integration/authentication/pat/PATWithAllSchemesTest.java

```diff
@@ -0,0 +1,259 @@
+import static org.zowe.apiml.util.requests.Endpoints.*;
+
+public class PATWithAllSchemesTest {
+
+    private static URI URL;
+    private static URI URL_SAF;
+    private static URI URL_PASS;
+
+    @BeforeAll
+   static void init() throws Exception {
+        SslContext.prepareSslAuthentication(ItSslConfigFactory.integrationTests());
+        URL = HttpRequestUtils.getUriFromGateway(ZOWE_JWT_REQUEST);
+        URL_SAF = HttpRequestUtils.getUriFromGateway(SAF_IDT_REQUEST);
+        URL_PASS = HttpRequestUtils.getUriFromGateway(REQUEST_INFO_ENDPOINT);
+    }
+
+    @Nested
+    class GivenPATWithZowejwtscheme {
+        @Test
+        @InfinispanStorageTest
+        void requestWithPATinPATHeader() {
+            Set<String> scopes = new HashSet<>();
+            scopes.add("zowejwt");
+            String pat = personalAccessToken(scopes);
+
+            verifyZoweZwtHeaders(given()
+                .config(SslContext.tlsWithoutCert)
+                .header(ApimlConstants.PAT_HEADER_NAME, pat)
+                .when()
+                .get(URL),pat);
+        }
+
+        @Test
+        @InfinispanStorageTest
+        void requestWithPATinAuthorizationHeader() {
+            Set<String> scopes = new HashSet<>();
+            scopes.add("zowejwt");
+            String pat = personalAccessToken(scopes);
+
+            verifyZoweZwtHeaders(given()
+                .config(SslContext.tlsWithoutCert)
+                .header(HttpHeaders.AUTHORIZATION,
+                    ApimlConstants.BEARER_AUTHENTICATION_PREFIX + " " + pat)
+                .when()
+                .get(URL),pat);
+        }
+
+        @Test
+        @InfinispanStorageTest
+        void requestWithPATinPATCookieHeader() {
+            Set<String> scopes = new HashSet<>();
+            scopes.add("zowejwt");
+            String pat = personalAccessToken(scopes);
+
+            verifyZoweZwtHeaders(given()
+                .config(SslContext.tlsWithoutCert)
+                .cookie(ApimlConstants.PAT_COOKIE_AUTH_NAME, pat)
+                .when()
+                .get(URL),pat);
+        }
+
+        @Test
+        @InfinispanStorageTest
+        void requestWithPATinCookieHeader() {
+            Set<String> scopes = new HashSet<>();
+            scopes.add("zowejwt");
+            String pat = personalAccessToken(scopes);
+
+          verifyZoweZwtHeaders(given()
+                .config(SslContext.tlsWithoutCert)
+                .cookie(SecurityUtils.COOKIE_NAME, pat)
+                .when()
+                .get(URL),pat);
+        }
+
+        void verifyZoweZwtHeaders(Response response,String pat) {
+            String basic = "Basic " + Base64.getEncoder().encodeToString((USERNAME + ":" + PASSWORD).getBytes());
+            assertEquals(response.getStatusCode(), HttpStatus.SC_OK);
+            assertNotEquals(response.getBody().path("headers.authorization"),pat);
+            assertNotEquals(response.getBody().path("headers.authorization"), basic);
```

Comment:
What is the reason to check if basic authentication is not set? What about checking if there is a JWT?